swagger.yml

```
@@ -1434,6 +1437,25 @@ definitions:
       bausparvertragSaldo:
         $ref: '#/definitions/euro'
 
+  angebotAlternative:
+    type: object
+    properties:
+      bezeichnung:
+        description: "Beispiel: Alternative: Downsell"
```

dass hier "Alternative: Downsell" als Standard gesetzt werden soll ist kaum erkennbar.

Wenn `bezeichnung` optional wäre, könnte man dass dann auch so formulieren
```suggestion
        description: "Kurze Bezeichnung für das alternative Angebot. Bei keiner Angabe wird die Standard Bezeichung (Alternative: Downsell) verwendet.
```